services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -231,15 +242,22 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
         <Row gutter={[16, 8]}>
           {system === SystemFlagEnum.core && (
             <Col span={24}>
-              <Typography.Paragraph>What is the type of the report?</Typography.Paragraph>
-              <Radio.Group
+              <Field
+                name="report_type"
+                id="report_type"
+                required
                 disabled={true}
```

Comment:
I set disabled to true here because at the moment in Core we don't have the ability for the user to select a permit condition category code if they select PRR, my understanding is that will be worked on in 5781.